Libs/Core/ctkUtils.cpp

```diff
@@ -79,6 +79,26 @@ void ctk::stlVectorToQList(const std::vector<std::string>& vector,
   std::transform(vector.begin(),vector.end(),std::back_inserter(list),&QString::fromStdString);
 }
 
+//------------------------------------------------------------------------------
+QSet<QString> ctk::qListToQSet(const QStringList& list)
```

Comment:
Since the input is a QStringList (not any list), the method name should be qStringListToSet.